src/main/java/ai/pluggy/client/request/ConnectorsSearchRequest.java

```diff
@@ -19,15 +21,16 @@ public ConnectorsSearchRequest(String name, List<String> countries) {
     this(name, countries, Collections.emptyList());
   }
 
-  public ConnectorsSearchRequest(String name, List<String> countries, List<String> types) {
+  public ConnectorsSearchRequest(String name, List<String> countries, List<ConnectorType> types) {
```

Comment:
@Gabrielpanga please validate if we are OK setting this PR to be released as v0.5.1, due to the _subtle_ breaking change.
I'd say that per business plan it'd be fine if we think of this as a "fix of an incomplete v0.5 release". But if we wanted to be more technically correct we maybe should re-version as v0.6.0, or else move forward with v0.5.1 but be sure to explicitly notify the users of this breaking change to mitigate the impact. In this case a note in the Release notes would be OK or maybe also send a slack message/email?